packages/jsii-python-runtime/src/jsii/_kernel/__init__.py

```diff
@@ -36,6 +40,12 @@ class Object:
     __jsii_type__ = "Object"
 
 
+def _handle_callback(kernel, callback):
```

Comment:
Note, this doesn't actually handle everything, this is one of the things that we need to coordinate with the sync callbacks with over who wants to "own" doing the complete implementation of this.

Reply:
Do you just want to merge this version and then I can do a PR with the updated version?

Reply:
That works for me.